challengeutils/discussion.py

```diff
@@ -1,114 +1,193 @@
-import synapseclient
-import os
-from synapseclient.exceptions import *
-
+'''
+Functions that interact with the Synapse discussion API
```

Comment:
need a period at the end. #nitpick

Reply:
should be a descriptive.
```suggestion
Interact with Synapse discussion API endpoints.
```